### PR TITLE
add: Authentication

### PR DESCRIPTION
**Pull request by kavyakunder**
Made signup login and logout forms:
login form
login ecommerce
logout form
logout ecommerce
signup form
signup ecomerce

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me . Good work Kavya on the design of these pages.

File: css/login-signup.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  padding: 2px;
+  color: var(--dark-grey);
+}
+.options {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 5px 0;
+}
+button.signin-btn {
```
Comment (shivampandey0) [inline css/login-signup.css]: - Any Specific reason to use `button.signin-btn` selector here? Why not just `signin-btn` ?

- `signin-btn` class is used for both SignIn and SignUp buttons. class name should be more generic in this case, right now it looks like it's only for SignIn Button

**Review by shivampandey0 — COMMENTED** [review]
The code looks clean and understandable.
- Login and signup button looks right-aligned, is it by design or they should be in the center?
- Added some suggestions

File: css/login-signup.css (hunk 1 above)
Comment (kavyakunder) [reply css/login-signup.css]: Hey I have made the changes suggested by you, please do have a look...

Comment (shivampandey0) [reply css/login-signup.css]: Good to merge.

**Pull request merged**